mint-choc-chip-skyblade/sshd-rando
# Randomize the item dropped by the Knight Academy Bell

## What does this PR do?
Adds the `Upper Skyloft - Ring Knight Academy Bell` check

## How do you test this changes?
* I tested ringing the bell, not collecting the item, reloading the area, and ringing the bell again to make sure the item dropped again.
* I tested ringing the bell, collecting the item, reloading the area, and ringing the bell again to make sure that the item didn't get dropped again.
* I tested ringing the bell and waiting a while to make sure the item doesn't despawn
